### ISSUE TITLE
Connector: Pinecone Vector DB

### ISSUE THREAD

**Issue by himanshu007-creator** [swirlai/swirl-search#674]
Hey, 
i would like to propose and work on a connector for pinecone vector database. Would plan it out with dev team and get started

**Comment by himanshu007-creator** [comment]
Yes, for various AI based companies, its often a usecase to store most likeable user queries and responses in vector database like pinecone or something else and later on on every subsequent user query, query generative Ai models as well as their own database for answers. This allows faster and consistent reponse. Not sure how we will exactly proceed with this, but can plan out on this
@sid-swirl

**Comment by sid-swirl** [comment]
Got it. The connector should be easy - the API looks fairly straightforward. The question is how to deliver the vector input? Do you have an example of a query you want to run today, via Swirl?

**Comment by sid-swirl** [comment]
BTW, you might take a look at Vespa https://github.com/swirlai/swirl-search/issues/692
